Code/autopkg

```diff
@@ -1801,13 +1803,14 @@ def make_override(argv):
 
     override_file = os.path.join(override_dir, "%s.recipe" % override_name)
     if os.path.exists(override_file):
-        log_err("An override plist already exists at %s, "
-                "will not overwrite it." % override_file)
-        return -1
-    else:
-        FoundationPlist.writePlist(override_plist, override_file)
-        log("Override file saved to %s" % override_file)
-        return 0
+        if not options.force:
+            log_err("An override plist already exists at %s, "
+                    "will not overwrite it." % override_file)
```

Comment:
Maybe now add the hint "Add --force option to force an overwrite." to the message.